src/tests/unit/tests/background/stores/global/permissions-state-store.test.ts

```diff
@@ -0,0 +1,76 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+import { PermissionsStateActions } from 'background/actions/permissions-state-actions';
+import { PermissionsStateStore } from 'background/stores/global/permissions-state-store';
+import { StoreNames } from 'common/stores/store-names';
+import { PermissionsStateStoreData } from 'common/types/store-data/permissions-state-store-data';
+import { StoreTester } from 'tests/unit/common/store-tester';
+
+describe('PermissionsStateStoreTest', () => {
+    const expectedDefaultStoreData: PermissionsStateStoreData = {
+        hasAllPermissions: false,
+    };
+
+    test('constructor', () => {
```

Comment:
name should be `'constructor, no side effects'`... the **no side effects** part is the most important one of this test. Although, I'm not sure we should care that much anymore. If you have more questions about the why of this test, please ask me.